person.rb

```diff
@@ -1,7 +1,8 @@
 require './nameable'
+require './rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindy move the rentals to `attr_accessor, the rentals should have both Getters and Setters.